resources/templates/messages.html

```diff
@@ -0,0 +1,34 @@
+<html>
+<head>
+    <link href="homepage.css" rel="stylesheet" type="text/css">
+</head>
+<body>
+<nav>
+    <h1 class="loginMessage">Welcome to borf</h1>
+    <h1 class="smalljoke">powered by rettiwt</h1>
+    <ul class="topBar">
+        <li>
+            <a title="logout" href="logout.html">Logout</a>
+        </li>
+        <li>
+            <a title="viewProfile" href="viewProfile.html">View my profile page</a>
+        </li>
+    </ul>
+</nav>
+
+<form action="/" method="post">
+        BLOG ABOUT YOURSELF
+</form>
+
+<form action="/create-message" method="post">
+    <input type="text" placeholder="Tell the world your borf" name="blogPost" width="1000"/>
+    <button type="submit">Submit</button>
+</form>
+
+<ul>
+    {{#messages}}
+    <li>{{post}} - made by {{creator}} <form action="/delete-entry" method="post"><input type="hidden" value="{{id}}" name="postId"><<button type="submit" name="delete">delete</button></form></li>
+    {{/messages}}
```

Comment:
Post and creator display the wrong way.  You pass them in out of order in your SelectAllMessages method.